metroae

```diff
@@ -678,38 +678,64 @@ function destroy {
 function pull {
     debug ${FUNCNAME[0]}
     if [[ ! -z $1 ]]; then
-        MAX_CONTAINER_VERSION=$1
-    fi
-    debug "${FUNCNAME[0]}: MAX_CONTAINER_VERSION: $MAX_CONTAINER_VERSION"
+        if [ $1 == "--repo" ]; then
+            REPO=$2
+            [[ "${REPO}" != */ ]] && REPO="${REPO}/"
+            DOCKER_REPO=${REPO}${DOCKER_REPO}
+            MAX_CONTAINER_VERSION=$3  
+            METRO_AE_IMAGE="$DOCKER_REPO/metroae"
```

Comment:
This line should go outside of the if/else block no? Because it won't be executed in the else